pkg/resourcemanager/webhook/seccompprofile/handler.go

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Similar to https://github.com/gardener/gardener/pull/6251#discussion_r913521466: you could add an envtest for the new webhook.